.rubocop/rails.yml

```diff
@@ -18,4 +18,4 @@ Rails/UnknownEnv:
     - test
 
 Rails/I18nLazyLookup:
-  Enabled: false
+  Enabled: true
```

Comment:
Since an enabled cop is the default, we can remove those lines completely (keeping one empty line at file end).